docs/methods/user-setting.md

```diff
@@ -0,0 +1,22 @@
+# Manage user settings
+
+> You must expose app data endpoints before calling these methods.
+
+Write to and read from [user settings](https://www.transposit.com/docs/building/user-config/#user-settings-schema) on behalf of the signed-in user. These methods only work for apps using sign-in.
+
+User settings rely on a pre-configured schema. Use [stash](./stash.md) if you want a schemaless key-value store.
+
+```javascript
+await transposit.userSetting.put("key-in-schema", "some-value");
+await transposit.userSetting.get("key-in-schema"); // "some-value"
+await transposit.userSetting.get("not-in-schema"); // <--- error!
```

Comment:
1. Do we want to add a line about how null resets the user to the default value?
2. Do we want to add a line about how setting the wrong type of value throws an error?